### PR TITLE
[PLAT-540] Lowered some log messages level

### PR DESCRIPTION
**Pull request by vgarvardt**

### CODE REVIEW

File: pkg/web/handlers.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func RecoveryHandler(w http.ResponseWriter, r *http.Request, err interface{}) {
 	switch internalErr := err.(type) {
 	case *errors.Error:
 		log.WithFields(log.Fields{"code": internalErr.Code, "error": internalErr.Error()}).
-			Warning("Internal error hadled")
+			Info("Internal error handled")
```
Comment (rafaeljesus) [inline pkg/web/handlers.go:31]: Should't this be `Error`?

**Comment by zeelax** [comment]
+1111111111111111111111
:heart: :heart_eyes: :kissing_heart: !!!!!!

[Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/157/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/157/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway)

File: pkg/web/handlers.go (hunk 1 above)
Comment (vgarvardt) [reply pkg/web/handlers.go:31]: This is handled error that we `panic()`, so it is logged with proper level before throwing. `Error` is something that really requires attention.

**Comment by zeelax** [comment]
:bitteschnell:

**Comment by italolelis** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/157/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/157/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/157/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway)

**Comment by kieranajp** [comment]
:shipit:

[Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/157/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/157/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/157/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway)

**Pull request merged**